README.md

```diff
@@ -38,6 +38,7 @@ Vars in this section directly correspond to the args available to the
 * `rhsm_server_proxy_password` - HTTP proxy password
 * `rhsm_auto_attach` - automatically consume from available subscriptions if registration succeeds (bool, default false, requires ansible >= 2.5)
 * `rhsm_environment` - Register with a specific environment in the destination org. (Used with Red Hat Satellite 6 or Katello)
+* `rhsm_pool` - specific pool name to consume (regex can be used as input)
```

Comment:
The admonition from the [redhat_subscrition](https://docs.ansible.com/ansible/latest/modules/redhat_subscription_module.html) module docs should probably be included here: "Use pool_ids instead if possible, as it is much faster." Additionally, `rhsm_pools` needs to indicate that it is now mutually exclusive with `rhsm_pool_ids`, and `rhsm_pool_ids` needs to indicate that it is now mutually exclusive with `rhsm_pools`